dobson/utils.py

```diff
@@ -0,0 +1,62 @@
+import configparser
+import json
+import os
+import subprocess
+from collections import namedtuple
+
+
+Device = namedtuple('Device', ['presence', 'user', 'model'])
+
+config = configparser.ConfigParser()
+config.read('dobson/config.ini')
+
+
+SLACK_API_TOKEN = config['dobson']['SlackApiToken']
+
+# Only allow querying for presence in certain channels
+ALLOWED_CHANNEL_IDS = json.loads(config['dobson']['AllowedChannelIds'])
+
+# The user ID of the dobson bot
+DOBSON_SLACK_ID = config['dobson']['DobsonSlackId']
+
+# file to log current MAC Addresses in
+MAC_LOG_FILE = config['dobson']['MacAddressLogFile']
+
+# create KNOWN_DEVICES, which maps mac addresses (str) to Device instances
+if not os.path.isfile(config['dobson']['KnownDevicesFile']):
+    with open(config['dobson']['KnownDevicesFile'], 'w') as f:
+        f.write('{}')
```

Comment:
This is to allow starting with a blank config and adding more users/macs through Slack, right? I'm not sure how I feel on this, since I feel like a config path could be typo'd and then it would have no data but would still work. That being said, this does more make sense to have when just setting up the project without any devices already stored...

Reply:
Hmm, fair enough that failing here could be more useful (since it's easy to add the file, but could be tough to debug why no users are showing up). Do you think we should change it to error with a message, maybe something like "Missing devices file `dobson/devices.json`" if the file doesn't exist?

Reply:
Yeah, I think that would be reasonable, and maybe saying that if they don't want any data, to just make the file with `{}` in it or something? I think then at least it makes it clear that they have no data to start with. I'd be ok with leaving it like this too though, I just thought it could be a problem if the device file path is typo'd or something

Reply:
Makes sense, I'll change it to that behavior.